tests/integration.js

```diff
@@ -130,7 +132,8 @@ function getInjectedData(path) {
   var url = urlResolve(process.env.ROOT_URL, path);
   var res = HTTP.get(url);
 
-  var matched = res.content.match(/data">(.*)<\/script/);
+  var matched = res.content.match(/data">(.*?)<\/script/);
```

Comment:
The query makes it non-greedy. Previously it was matching additional tags. See http://regexr.com/3cu6c